source/_docs/vanity-domains.md

```diff
@@ -36,6 +36,18 @@ At your DNS provider, create a wildcard CNAME record pointing to our edge. If yo
 
  - **Environment URLs are permanent.** If an organization is removed as the supporting organization, any environment created during its association will keep the original URL after removal. Paid sites can add custom domains to any environment, as a workaround for those wishing to use different URL's after launch and disassociation of the site with the organization.
 
+ - To ensure maximum uptime, configure DNS of Custom domains (e.g., `www.example.com` on Live) using the environment's Platform domain for the CNAME record value as recommended within the **<span class="glyphicons glyphicons-global"></span> Domains / HTTPS** page, never a Vanity domain:
```

Comment:
This may be one of the most densely packed sentences I've ever come across, and I'll need help deciphering it.